### PR TITLE
Feature/add a expect to fail wip flag

### PR DESCRIPTION
**Pull request by fliiiix**

### CODE REVIEW

File: docs/commandline.rst [hunk 1]
```diff
@@ -210,6 +210,20 @@ To learn how to tag Features and Scenarios please refer to :ref:`tutorial#tags`
 section.
 
 
+Run - Work in progress
+-------------------------------
+
+Radish is able change the exception of the outcome. Features which
```
Comment (timofurrer) [inline docs/commandline.rst]:
```suggestion
Radish is able change the state of the outcome. Features which
```

Comment (timofurrer) [inline docs/commandline.rst]:
```suggestion
Radish is able change the exception of the outcome. Scenarios which
```

File: docs/commandline.rst [hunk 2]
```diff
@@ -210,6 +210,20 @@ To learn how to tag Features and Scenarios please refer to :ref:`tutorial#tags`
...
+are still work in progress and are expected to fail, can be run with:
+
+.. code:: bash
+
+   radish SomeFeature.feature --wip
+
+To count as a success all Scenarios in this Feature need to fail.
+If a Scenario passes the run is failed.
```
Comment (timofurrer) [inline docs/commandline.rst]: Add hint about combining with `--tags 'not wip'` and `--tags 'wip'` and their semantic meaning ..

Comment (fliiiix) [reply docs/commandline.rst]: Do you mean something like a recommended workflow?

Comment (fliiiix) [reply docs/commandline.rst]: check out the updated version

**Pull request merged**